SourceGitphp/SourceGitphp.php

```diff
@@ -0,0 +1,367 @@
+<?php
+
+# Copyright (c) 2012 John Reese
+# Licensed under the MIT license
+
+if ( false === include_once( config_get( 'plugin_path' ) . 'Source/MantisSourcePlugin.class.php' ) ) {
+	return;
+}
+
+require_once( config_get( 'core_path' ) . 'url_api.php' );
+
+class SourceGitphpPlugin extends MantisSourcePlugin {
+	public function register() {
+		$this->name = plugin_lang_get( 'title' );
+		$this->description = plugin_lang_get( 'description' );
+
+		$this->version = '0.18';
+		$this->requires = array(
+			'MantisCore' => '1.2.0',
+			'Source' => '0.16',
+		);
+
+		$this->author = 'John Reese';
+		$this->contact = 'john@noswap.com';
+		$this->url = 'http://noswap.com';
```

Comment:
He did 95% of it. I'm not sure I'm wanting to be the maintainer of record on this. Just thought someone else out there might find it useful.